js/actions/actions.js

```diff
@@ -0,0 +1,31 @@
+var AppDispatcher = require('../dispatcher/app_dispatcher');
+var ActionTypes = require('../constants/constants').ActionTypes;
+var UserStore = require('../stores/user_store')
+
+module.exports = {
+  createMessage: function(text) {
+    AppDispatcher.dispatch({
+      type: ActionTypes.NEW_MESSAGE,
+      text: text,
+      user: UserStore.localUser().name,
+      timestamp: new Date().getTime()
```

Comment:
i have the impression that all data other than `type` should be nested in a `data` object.. what do you think?